src/Microsoft.Health.Fhir.Anonymizer.Shared.FunctionalTests/ResourceTests.cs

```diff
@@ -88,6 +89,22 @@ public void GivenAPatientResource_WhenAnonymizingWithMultipleSubstituteConfig_Th
             FunctionalTestUtility.VerifySingleJsonResourceFromFile(engine, ResourceTestsFile("patient-substitute-multiple-2.json"), ResourceTestsFile("patient-substitute-multiple-2-target.json"));
         }
 
+        [Fact]
+        public void GivenAPatientResource_WhenAnonymizingWithProcessingError_IfSkip_NullResultWillBeReturned()
+        {
+            AnonymizerEngine engine = new AnonymizerEngine(Path.Combine("Configurations", "configuration-skip-processing-error.json"));
+            string testContent = File.ReadAllText(ResourceTestsFile("patient-basic.json"));
+            Assert.Null(engine.AnonymizeJson(testContent));
+        }
+
+        [Fact]
+        public void GivenAPatientResource_WhenAnonymizingWithProcessingError_IfRaise_ExceptionWillBeThrown()
+        {
+            AnonymizerEngine engine = new AnonymizerEngine(Path.Combine("Configurations", "configuration-raise-processing-error.json"));
+            string testContent = File.ReadAllText(ResourceTestsFile("patient-basic.json"));
+            Assert.Throws<AnonymizerProcessFailedException>(() => engine.AnonymizeJson(testContent));
+        }
+
```

Comment:
Can we add a test when users do not configure "processingErrors"? The default behavior is raise right?

Reply:
Added it, thanks.